tests/buyProduct_test.js

```diff
@@ -0,0 +1,59 @@
+const USER = {
+    email: "yulialvju@gmail.com",
+    password: "Use54321!",
+    firstName: "Julia",
+    lastName: "Leonova",
+    phone: "+380932849875",
+    address: "Yesenina, 11",
+    city: "Kharkiv",
+    postcode: "61072",
+}
+
+
+Feature('purchase');
+
+Scenario.only('buy product',  async ({ I, homePage, checkoutPage, productPage }) => {
+   
+    I.login(USER);
+    homePage.clickDropdownCartIcon();
+    let numOfElements = await I.grabNumberOfVisibleElements('//i[@class="linearicons-trash"]');
+    if(numOfElements) {
+        homePage.clickRemoveItems();
+    };
+    
+    I.amOnPage('http://opencart.qatestlab.net/index.php?route=product/product&product_id=74');
+    productPage.clickSelectField();
+    productPage.clickColor();
+    productPage.clickAddToCartButton();
+    homePage.clickDropdownCartIcon();
+    homePage.clickCheckout();
+    checkoutPage.fillCheckoutForm2(USER);
+    checkoutPage.clickCountryToggle();
+
+    for (let i = 0; i < 4; i++) {
```

Comment:
don't hardcode cycle to 4, use `while` cycle with condition: button is visible or similar

Reply:
> don't hardcode cycle to 4, use `while` cycle with condition: button is visible or similar

Added. But then the Agree button is not clicked, but 4th ContinueButton was